### ISSUE TITLE
[BUG] bridge. never updates files in com.mojang

### ISSUE THREAD

**Issue by dombomb64** [bridge-core/editor#39]
**Summary**
When editing a file, bridge. doesn't update it in the com.mojang folder.

**To Reproduce**
1. Set up and open bridge.
2. Allow permission to data and com.mojang folders
3. Select an addon
4. Edit a file

**Observed behavior**
The file updates in the projects folder, but doesn't update in the com.mojang folder.
**Expected behavior**
The file updates in both folders.

**Screenshots / File Attachments**
Projects Folder:
![donut_block json - Notepad 7_19_2021 2_13_19 PM](https://user-images.githubusercontent.com/49823019/126207619-d829ef6b-ec69-4f3a-a49b-7d3c7c16e10a.png)
com.mojang Folder:
![donut_block json - Notepad 7_19_2021 2_14_58 PM](https://user-images.githubusercontent.com/49823019/126207637-18e003de-5380-49f8-ac3d-4ce5b5f4fd4d.png)

**Platform:**
 - OS: Windows 10
 - App Version: v2.0.0

**Additional context**
 - The addon I selected was originally created with bridge. v1.7, I think. I don't know if that has anything to do with it.
 - I know that it doesn't work with behavior packs or resource packs, but I don't know if it works with skin packs because I can't find out how to do skin packs with bridge.

**Comment by Joelant05** [comment]
It looks like the file in your com.mojang is the wrong project because it still has the bridge file version comment.

**Comment by dombomb64** [comment]
> It looks like the file in your com.mojang is the wrong project because it still has the bridge file version comment.

Yes, this is true.

> If you transfered the project through bridge. v1 the pack should end with "V2" in the com.mojang folder

That’s strange, only one of my packs is like that, and it’s the first one (in alphabetical order). All of the packs in the development folders were created with bridge. v1, though, including the one that got changed for v2.

**Comment by solvedDev** [comment]
Did you transfer all of your projects to bridge. v2? If so, in order for them to also appear inside of the com.mojang folder, you have to load them up at least once inside of v2

**Comment by dombomb64** [comment]
> Did you transfer all of your projects to bridge. v2? 
I moved the old bridge. v1 projects out of the com.mojang folder, and all of the projects except the one I was struggling with started working again, I think. As for the one that I was struggling with, I tried to delete the .bridge folder so it would create a new one, but even though it did create a new one, it still wouldn’t sync the project over to the com.mojang folder. I ended up making a new project and moving the files over.
